ros/src/util/packages/runtime_manager/scripts/simulation.yaml

```diff
@@ -67,6 +75,14 @@ params :
       cmd_param :
         only_enable : True
         dash        : '--'
+    - name      : other
+      label     : 'Other flags:'
+      kind      : str
+      v         : '                        '
```

Comment:
@alexandrx why these blank spaces?
when I try to play is causing the playback to never start.

Reply:
The blank spaces were to force the textfield to render with some minimum length.
It does play! Even if you keep blank spaces there is no problem.